frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -273,6 +281,7 @@ export class PieceEditor implements OnInit {
       addingPositions: [],
       deletePositions: [],
     };
+    this.superDisplay.verifyLoad(newPiece.uuid, newPiece.isOpen);
```

Comment:
This would be more readable: 
```
if (newPiece.type === 'SUPER') {
  this.balletDisplayService.openSuperBallet(newPiece.uuid);
}
```

Reply:
With the new naming this is now:

this.superBalletDisplay.verifyIsInDisplayList(newPiece.uuid, newPiece.isOpen);

It is important that already displayed super ballet don't change their open state when they are being re-read from the database, which happens after each save of a new or edited ballet. Because of this, the logic of the verifyIsInDisplayList is that ballets already in the display list are left untouched. However, ballets not in the display list are added in two ways. Ballets from the database go in as closed. Newly created ballets go in as open. (The customers can change their name about this, but at this point they have asked for the open state.)